### DISCUSSION TITLE
Error:   Forbidden for url on startup

### DISCUSSION THREAD

**Discussion by sierraworks** [mrlt8/docker-wyze-bridge#1023] [Q&A] [answered]
When trying to start the latest bridge with the following command:

$ 
docker run \
  -e WYZE_EMAIL=xxx@gmail.com \
  -e WYZE_PASSWORD=xxx! \
  -e API_KEY=xxx \
  -e API_ID=xxx \
  -p 1935:1935 -p 8554:8554 -p 8888:8888 -p 5000:5000 \
  mrlt8/wyze-bridge:latest

I get the following:

2023/10/25 20:25:51 [WyzeBridge] � STARTING DOCKER-WYZE-BRIDGE v1.7.4

2023/10/25 20:25:51 [WyzeBridge] � Could not find local cache for 'user'
2023/10/25 20:25:51 [WyzeBridge] � Could not find local cache for 'auth'
2023/10/25 20:25:51 [WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
2023/10/25 20:25:51 [WyzeBridge]  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://172.17.0.2:5000 (Press CTRL+C to quit)
2023/10/25 20:25:51 [WyzeBridge] 403 Client Error: Forbidden for url: https://auth-prod.api.wyze.com/user/login

The only meta character I have is a ! in the password string so I don't think it's that.   I am using 2 factor but also have the API ID/Key.

This is on a mac with docker:

$  docker --version
Docker version 20.10.17, build 100c701

Thanks for any help.

**Comment by mrlt8** [comment, marked as answer]
Wyze made some changes to their API earlier this year, and depreciated the old authentication url `https://auth-prod.api.wyze.com/user/login` back in July.

You will need to update your bridge to v2.3.x or newer for compatibility with the latest changes to the Wyze API.

You can try pulling the latest image with:

```
docker pull mrlt8/wyze-bridge:latest
```
you should then be able to use your usual docker run command.

**Reply by sierraworks** [reply]
Many thanks,   I could have sworn that I had the latest but this did fix the issue.
